index.js

```diff
@@ -61,7 +61,7 @@ Gists.prototype.list = function(options, cb) {
  */
 
 Gists.prototype.all = function(opts, cb) {
-  this.getAll('/gists/', opts, cb);
+  this.pages('/gists', opts, cb);
```

Comment:
typo, the method name is `.paged`, not `.pages`.

Reply:
Thanks @jonschlinkert. What do you think keep using `.get` with passing in a param `per_page` instead? Which request to change on`github-base`. Something like this, https://github.com/jonschlinkert/github-base/pull/17